search-index.js

```diff
@@ -0,0 +1,106 @@
+const lunr = require('lunr');
+const redis = require('redis');
+const { promisify } = require('util');
```

Comment:
where does this import come from? I can't see util.js in the PR nor in the package.json

Reply:
util is part of Node.js https://nodejs.org/api/util.html.

Reply:
I wish there was an "I'm embarrassed I asked" emoji :-)